doc/source/categorical.rst

```diff
@@ -766,6 +766,14 @@ Dtype comparisons work:
     dtype == np.str_
     np.str_ == dtype
 
+To check if a Series contains Categorical data, with pandas 0.16 or later, use
+``hasattr(s, 'cat')``:
+
+.. ipython:: python
```

Comment:
ah, yes, sorry, I was looking too quickly, thought you were giving an example of that accessing the attribute raises, but it is `hasattr`, not `getattr` ..